README.md

```diff
@@ -320,6 +334,11 @@ The following settings are set for you automatically by [pluto](sources/api/) ba
 * `settings.kubernetes.cluster-dns-ip`: The CIDR block of the primary network interface.
 * `settings.kubernetes.node-ip`: The IPv4 address of this node.
 * `settings.kubernetes.pod-infra-container-image`: The URI of the "pause" container.
+* `settings.kubernetes.kube-reserved`: Resources reserved for node components.
+  * Bottlerocket provides default values for the resources. Default values are taken from [EKS' calculations](https://github.com/awslabs/amazon-eks-ami/blob/db28da15d2b696bc08ac3aacc9675694f4a69933/files/bootstrap.sh):
+    `cpu`: in millicores from the total number of vCPUs available on the instance.
```

Comment:
If you intend for these to be bullets, you need to add a "*" in front of each bullet point. You can check the richdiff by clicking on the page icon in the upper right corner when viewing diffs.

Reply:
yeah. I will fix in next push